privacy/ccpa/policy_test.go

```diff
@@ -303,38 +546,74 @@ func TestValidateConsent(t *testing.T) {
 func TestShouldEnforce(t *testing.T) {
 	testCases := []struct {
 		description string
+		bidder      string
 		policy      Policy
 		expected    bool
 	}{
 		{
 			description: "Enforceable",
+			bidder:      "a",
 			policy:      Policy{Value: "1-Y-"},
 			expected:    true,
 		},
+		{
+			description: "Enforceable - No Sale For Different Bidder",
+			bidder:      "a",
+			policy:      Policy{Value: "1-Y-", NoSaleBidders: []string{"b"}},
+			expected:    true,
+		},
 		{
 			description: "Not Enforceable - Not Present",
+			bidder:      "a",
 			policy:      Policy{Value: ""},
 			expected:    false,
 		},
 		{
 			description: "Not Enforceable - Opt-Out Unknown",
+			bidder:      "a",
 			policy:      Policy{Value: "1---"},
 			expected:    false,
 		},
 		{
 			description: "Not Enforceable - Opt-Out Explicitly No",
+			bidder:      "a",
 			policy:      Policy{Value: "1-N-"},
 			expected:    false,
 		},
+		{
+			description: "Not Enforceable - No Sale All Bidders",
+			bidder:      "a",
+			policy:      Policy{Value: "1-Y-", NoSaleBidders: []string{"*"}},
+			expected:    false,
+		},
+		{
+			description: "Not Enforceable - No Sale All Bidders Mixed With Specific Bidders",
```

Comment:
Should this be allowed? Or should specifying other values with a * element be an error? I think the "nicest" way to handle it is as written in this test.

Reply:
I agree